css/homepage.scss

```diff
@@ -50,6 +50,20 @@
     }
   }
 }
+#diff-data-api {
+  h2 {
+      max-width: 60em;
+      margin: 1em auto;
+      font-size: 150%;
+  }
+  .item {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    text-align: justify;
+  }
```

Comment:
All this is redundant with Semantic UI `grid` variations. Why do we need it? Are you sure it doesn't break the responsive layout?